src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorDataflowIntegrationTest.java

```diff
@@ -15,19 +15,29 @@
 
 /**
  *
- * To run the cloud tests, you need:
- * (1) a client-secrets.json in the project's base folder
- * (2) set the TEST_PROJECT and TEST_STAGING_GCS_FOLDER environment variables
- * (3) set the HELLBENDER_TEST_INPUTS environment variable to a GCP folder that has the test inputs this needs
- *     (from src/test/resources/org/broadinstitute/hellbender/tools/BQSR/, see test code
- *      to see which specific files are used)
- *     e.g. gs://MYBUCKET/TESTINPUTS/
+ * To run the cloud tests, you need the following environment variables:
+ * HELLBENDER_TEST_PROJECT - the short name of your Google Cloud Platform project
+ * HELLBENDER_TEST_APIKEY - the API key associated with that project
+ * HELLBENDER_TEST_STAGING - a Google Cloud Storage folder to hold temporary files, e.g. gs://MYBUCKET/staging/
+ * HELLBENDER_TEST_INPUTS - a Google Cloud Storage path (ending in "/") that contains this folder:
+ *                          org/broadinstitute/hellbender/tools/dataflow/BaseRecalibratorDataflow/
+ *                          with the BaseRecalibratorDataflow test inputs.
+ *                          The files must be shared publicly (i.e. world-readable).
+ *                          The specific files used for this test are:
+ *                          - human_g1k_v37.chr17_1Mb.*
+ *                          - NA12878.chr17_69k_70k.dictFix.*
  */
 public final class BaseRecalibratorDataflowIntegrationTest extends CommandLineProgramTest{
 
-    private final static String CLOUD_INPUTS = System.getenv("HELLBENDER_TEST_INPUTS");
-    private final static String TEST_PROJECT = System.getenv("TEST_PROJECT");
-    private final static String TEST_STAGING_GCS_FOLDER = System.getenv("TEST_STAGING_GCS_FOLDER");
+    // shortname of the project that stores the data and will run the code.
+    private final static String TEST_PROJECT = System.getenv("HELLBENDER_TEST_PROJECT");
+    // API key for TEST_PROJECT
+    private final static String API_KEY = System.getenv("HELLBENDER_TEST_APIKEY");
+    // A writeable folder on the project's GCS
+    private final static String TEST_STAGING_GCS_FOLDER = System.getenv("HELLBENDER_TEST_STAGING");
+    // A GCS path (on that project) where the test inputs are stored.
+    private final static String THIS_TEST_INPUTS = "org/broadinstitute/hellbender/tools/BQSR/";
+    private final static String CLOUD_INPUTS = System.getenv("HELLBENDER_TEST_INPUTS") + THIS_TEST_INPUTS;
```

Comment:
Please push all environment variables for running dataflow tests up into the `BaseTest` class so that we can enforce usage of a uniform set of variables across the test suite.